### PR TITLE
Refactored hello_world.

### PR DESCRIPTION
**Pull request by tanmaydas82**
BUG=http://b/277264885, http://b/279076022, http://b/280388095

### CODE REVIEW

File: tensorflow/lite/micro/examples/hello_world/README.md [hunk 1]
```diff
@@ -40,14 +40,14 @@ bazel run :evaluate_test
...
+bazel run tensorflow/lite/micro/examples/hello_world:hello_world_test
 ```
 And to run it using make 
 ```bash
-make -f tensorflow/lite/micro/tools/make/Makefile test_evaluate_cc_test
+make -f tensorflow/lite/micro/tools/make/Makefile test_hello_world_test
 ```
 
-The source for the test is [evaluate_test.cc](evaluate_test.cc).
+The source for the test is [evaluate_test.cc](hello_world_test.cc).
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/hello_world/README.md:50]:
```
[evaluate_test.cc](hello_world_test.cc) --> [hello_world_test.cc](hello_world_test.cc).
```

File: tensorflow/lite/micro/examples/hello_world/evaluate.py [hunk 2]
```diff
@@ -50,9 +50,22 @@ def invoke_tflite_interpreter(input_shape, interpreter, x_value, input_index,
...
+
+
+# Generate a list of 1000 random floats in the range of 0 to 2*pi.
+def generate_random_float_input(sample_count=1000):
+  # Generate a uniformly distributed set of random numbers in the range from
+  # 0 to 2π, which covers a complete sine wave oscillation
+  np.random.seed(42)
   x_values = np.random.uniform(low=0, high=2 * np.pi,
                                size=sample_count).astype(np.float32)
   # Shuffle the values to guarantee they're not in order
```
Comment (advaitjain) [inline tensorflow/lite/micro/examples/hello_world/evaluate.py:71]: let's remove the shuffle, not sure why we had this in the first place.

**Review by advaitjain — CHANGES_REQUESTED** [review]
apologies for the long delay here.

File: tensorflow/lite/micro/examples/hello_world/README.md (hunk 1 above)
Comment (tanmaydas82) [reply tensorflow/lite/micro/examples/hello_world/README.md:50]: done

File: tensorflow/lite/micro/examples/hello_world/evaluate.py [hunk 3]
```diff
@@ -50,7 +50,18 @@ def invoke_tflite_interpreter(input_shape, interpreter, x_value, input_index,
...
 
 # Generate a list of 1000 random floats in the range of 0 to 2*pi.
-def generate_random_input(sample_count=1000):
+def generate_random_int8_input(sample_count=1000):
+  # Generate a uniformly distributed set of random numbers in the range from
+  # 0 to 2π, which covers a complete sine wave oscillation
+  x_values = np.random.uniform(low=0, high=2 * np.pi,
+                               size=sample_count).astype(np.int8)
+  # Shuffle the values to guarantee they're not in order
+  np.random.shuffle(x_values)
```
[the remark this answers is not in the document]
Comment (tanmaydas82) [reply tensorflow/lite/micro/examples/hello_world/evaluate.py:60]: done

File: tensorflow/lite/micro/examples/hello_world/evaluate.py (hunk 2 above)
Comment (tanmaydas82) [reply tensorflow/lite/micro/examples/hello_world/evaluate.py:71]: done

**Review by advaitjain — APPROVED** [review]

**Pull request merged**